docs/content/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -46,12 +46,12 @@ nodes:
 - role: control-plane
   extraPortMappings:
   - containerPort: 8081
-    hostPort: 8081
+    hostPort: 8094
 ```
 
 For the guilder cluster, which will get two workloads, create a file
 named `guilder-config.yaml` with the following contents.  The workload
-that uses hostPort 8081 goes in both clusters, while the workload that
```

Comment:
The one does not change.  `hostPort` is kube networking jargon, referring to the host of the pod.  That is buried inside the kind cluster, it is not talking about the host of the kind cluster.